From ac8b5090a4d3c492a328d16b6256c53157715909 Mon Sep 17 00:00:00 2001
From: Kevin Jeon <kevinjeon@google.com>
Date: Fri, 17 Mar 2023 19:26:17 +0000
Subject: [PATCH] Grant MANAGE_USERS access to Traceur

This change updates the privapp allowlist to grant the MANAGE_USERS
permission to Traceur. This permission is needed to query admin user
status, as Traceur shouldn't be able to start if the current user is not
an admin.

Test: Using ABTD, apply this change with ag/22119816 to verify that
      Traceur still works as intended (opening app, tracing, etc.).
Bug: 262243665
Bug: 262244249
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:f42db15239663604eb5d36edb04a0f9a04576568)
Merged-In: I8e2174065b686c052cb080b3590ea4d89e7a7783
Change-Id: I8e2174065b686c052cb080b3590ea4d89e7a7783
---
 data/etc/privapp-permissions-platform.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index 813b7995fe89..3dd16721e529 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -516,6 +516,8 @@ applications that come with the platform
         <permission name="android.permission.STATUS_BAR"/>
         <!-- Permissions required to query Betterbug -->
         <permission name="android.permission.QUERY_ALL_PACKAGES"/>
+        <!-- Permissions required to query admin user status -->
+        <permission name="android.permission.MANAGE_USERS"/>
     </privapp-permissions>
 
     <privapp-permissions package="com.android.tv">
-- 
2.40.0.634.g4ca3ef3211-goog

